src/neofs_testlib/env/env.py

```diff
@@ -288,13 +290,17 @@ def generate_config_file(config_template: str, config_path: str, custom=False, *
         with open(config_path, mode="w") as fp:
             fp.write(rendered_config)
 
-    @staticmethod
-    def get_available_port() -> str:
-        s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-        s.bind(("", 0))
-        addr = s.getsockname()
-        s.close()
-        return addr[1]
+    @classmethod
+    def get_available_port(cls) -> str:
+        for _ in range(len(cls._busy_ports) + 2):
+            s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+            s.bind(("", 0))
+            addr = s.getsockname()
+            s.close()
```

Comment:
Frankly, I don't like this code. But I can't immediately suggest any better one.